packages/tools/devtools/devtools-view/src/DevtoolsView.tsx

```diff
@@ -444,6 +469,18 @@ function Menu(props: MenuProps): React.ReactElement {
 	}
 	return (
 		<Stack.Item styles={menuStyles}>
+			<div
+				style={{
+					minWidth: "250px",
+					display: "flex",
+					cursor: "pointer",
+					margin: "3px",
+				}}
+				onClick={onHomeClicked}
+			>
+				<h4 style={{ margin: "0px 3px 0px 0px" }}>Home</h4>
+				<Home20Regular />
```

Comment:
Personally I'd drop the icon; we won't have icons for every menu option and this one "clashes functionally" with the container refresh button right under it. The latter has a specific functionality, whereas this one is just a decoration and doesn't do anything different from clicking on the "Home" text.

I'd like to hear from the team on this though.

Reply:
I could go either way. I like icons generally. I actually think the placement of the container refresh button is somewhat confusing. It might be better for us to float that button to the right (or something), so it is differentiated from the menu item icons, and keep the Home icon. But that would be outside the scope of this PR. In the interim, I don't have a strong preference for keeping or removing the Home icon.